po/zh_CN.po

```diff
@@ -4276,123 +4275,125 @@ msgstr "员工"
 msgid ""
 "The owner %s will be deleted.\n"
 "Are you sure you want to do this?"
-msgstr "此科目尚未结算。您确实要结束？"
+msgstr ""
+"所有者 %s 将被删除。\n"
+"您确定要执行？"
 
 #. **********************************************************
 #. Actions
 #. **********************************************************
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:192
 #: ../gnucash/gnome/gnc-plugin-page-register.c:197
 msgid "Cu_t Transaction"
-msgstr "剪切交易(_T)"
+msgstr "剪切交易事项(_T)"
 
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:193
 #: ../gnucash/gnome/gnc-plugin-page-register.c:198
 msgid "_Copy Transaction"
-msgstr "复制交易(_C)"
+msgstr "复制交易事项(_C)"
 
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:194
 #: ../gnucash/gnome/gnc-plugin-page-register.c:199
 msgid "_Paste Transaction"
-msgstr "粘贴交易(_P)"
+msgstr "粘贴交易事项(_P)"
 
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:195
 #: ../gnucash/gnome/gnc-plugin-page-register.c:200
 msgid "Dup_licate Transaction"
-msgstr "复制交易(_L)"
+msgstr "创建交易事项副本(_L)"
 
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:196
 #: ../gnucash/gnome/gnc-plugin-page-register.c:201
 #: ../gnucash/gnome/gnc-split-reg.c:1293
 #: ../gnucash/gnome-utils/gnc-tree-control-split-reg.c:1064
 msgid "_Delete Transaction"
-msgstr "删除交易(_D)"
+msgstr "删除交易事项(_D)"
 
 #: ../gnucash/gnome/gnc-plugin-page-register2.c:197
 #: ../gnucash/gnome/gnc-plugin-page-register.c:205
 #, fuzzy
 msgid "Cu_t Split"
-msgstr "自动拆分"
+msgstr "剪切分录(_t)"
```

Comment:
I would consistently use uppercase latin letters for mnemonics. They are easier to read.

Reply:
Fixed.